tensorflow/tools/docker/Dockerfile.devel

```diff
@@ -38,6 +38,8 @@ RUN pip --no-cache-dir install \
         && \
     python -m ipykernel.kernelspec
 
+# RUN ln -s /usr/bin/python3 /usr/bin/python#
```

Comment:
Is it commented out on purpose?

Reply:
Yes, it is. It gets modified.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/docker/parameterized_docker_build.sh#L309